scripts/datacommons/place_cohorts/generate_cohort_set.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A script to generate a new place CohortSet, and its members based on a CSV."""
+
+import csv
+
+from absl import app
+from absl import flags
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string("csv", None, "Path to the raw csv containing the places.")
+flags.DEFINE_string("set_id", None, "DCID and name of the CohortSet.")
+flags.DEFINE_string(
+    "place_id_property", None,
+    "The CSV column with the place identifiers. It should also be the DCID of the property."
+)
+flags.DEFINE_string("place_type", None,
+                    "The DCID of the Place type of CohortSet members.")
```

Comment:
Sure, I am ok with that, but want to make sure you saw this comment: "This gets written as MCF though, so it's not just for resolution (gets written to graph). It would just add "Place" as a typeOf, which while not so bad, is not preferred?" ?

Reply:
Saw and responded :)

Reply:
Thanks!